eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -679,31 +678,6 @@ spec =
                       SNothing
                   )
           submitGovAction_ constitutionAction1
-    describe "Votes fail as expected" $ do
-      it "on expired gov-actions" $ do
-        modifyPParams $ ppGovActionLifetimeL .~ EpochInterval 2 -- Voting after the 3rd epoch should fail
-        khDRep <- setupSingleDRep
-        gaidConstitutionProp <- submitInitConstitutionGovAction
-        passEpoch
-        passEpoch
-        passEpoch
-        let voter = DRepVoter $ KeyHashObj khDRep
-        submitFailingVote
-          voter
-          gaidConstitutionProp
-          [ injectFailure $
-              VotingOnExpiredGovAction $
-                (voter, gaidConstitutionProp) NE.:| []
-          ]
-      it "on non-existant gov-actions" $ do
-        khDRep <- setupSingleDRep
-        gaidConstitutionProp <- submitInitConstitutionGovAction
-        let voter = DRepVoter $ KeyHashObj khDRep
-            dummyGaid = gaidConstitutionProp {gaidGovActionIx = GovActionIx 99} -- non-existant `GovActionId`
-        submitFailingVote
-          voter
-          dummyGaid
-          [injectFailure $ GovActionsDoNotExist $ dummyGaid NE.:| []]
```

Comment:
Why are these tests removed?

Reply:
They are duplicated just below, in the describe "Voting" test

Reply:
Oh, right